docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -65,7 +67,7 @@ insert into account values
   ('kevin','checking', 500);
 ```
 
-Now, we're going to perform a deposit and a withdrawal at the same time. Also, assume we want to give higher priority to deposit transactions (when compared to withdrawals). To simulate this, we're going to perform two operations concurrently - a withdrawal in one session, and a deposit from a separate session. The deposit transaction starts after the withdrawal is initiated, but occurs before the withdrawal is completed from a separate session. This is shown below.
+To set a transaction priority for concurrent transactions, perform a deposit and a withdrawal at the same time and set a higher priority to deposit transactions. To simulate this, perform the two operations concurrently - a withdrawal in one session, and a deposit from another. The deposit transaction starts after the withdrawal is initiated, but occurs before the withdrawal is completed from a separate session as demonstrated in the following table:
```

Comment:
Suggestion:

To set a transaction priority for concurrent transactions, perform a deposit and a withdrawal at the same time, and set a higher priority to deposit transactions. To simulate this, perform the two operations concurrently - a withdrawal in one session and a deposit in another session. The deposit transaction starts after the withdrawal has been initiated, but occurs before the withdrawal is completed from a separate session, as demonstrated in the following table: